src/view/screens/Home.tsx

```diff
@@ -93,7 +97,7 @@ function HomeScreenReady({
       setDrawerSwipeDisabled(index > 0)
       const feed = allFeeds[index]
       setSelectedFeed(feed)
-      persisted.write('lastSelectedHomeFeed', feed)
```

Comment:
Note: this still happens but is now taken care by `setSelectedFeed` from the provider.